README.md

```diff
@@ -1199,6 +1199,7 @@ Table of Contents
 
 ## Screenshot APIs
 
+  * [ProxyCrawl.com](https://proxycrawl.com/screenshots-api) - Automate website screenshots while avoiding captchas and blocks with an API.
```

Comment:
please elaborate on what is the free plan for this in the description

Reply:
@ripienaar done